### PR TITLE
feat: init ghost collection view

### PR DESCRIPTION
**Pull request by junbeomlee**

### CODE REVIEW

File: src/components/GhostCard.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<template>
```
Comment (hihiboss) [inline src/components/GhostCard.vue]: I think this file moved to some package like `src/components/cards`, `src/components/ghosts`, or etc.
How about you?

File: src/components/section/GhostDetailSection.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<template>
+  <div>
+    <h2>GhostDetailSection</h2>
+    <template v-if=ghost>
+      <h5>{{ghost.gene}}</h5>
```
[the remark this answers is not in the document]
Comment (junbeomlee) [reply src/components/section/GhostDetailSection.vue]: the ghost image is determined by gene!
so i think ghost does not need to have image!

File: src/components/GhostCard.vue (hunk 1 above)
Comment (junbeomlee) [reply src/components/GhostCard.vue]: `src/components/cards` looks better

**Comment by junbeomlee** [comment]
fixed it

File: src/components/section/GhostDetailSection.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (hihiboss) [reply src/components/section/GhostDetailSection.vue]: Don’t we need a code to load image like `<h5>image</h5>` in `GhostCard.vue`?

**Pull request merged**